examples/CMakeLists.txt

```diff
@@ -22,7 +22,9 @@ FOREACH(examples ${${PROJECT_NAME}_EXAMPLES_PYTHON})
     ${PYTHON_EXECUTABLE} -c "import ${examples}" \${INPUT})
 
   # examples are too slow in Debug mode to be used as tests
-  IF(CMAKE_BUILD_TYPE STREQUAL "Release")
+  IF(NOT CMAKE_BUILD_TYPE STREQUAL "Debug")
     ADD_PYTHON_UNIT_TEST("example-python-${examples}" "examples/${examples}.py" bindings/python)
-  ENDIF(CMAKE_BUILD_TYPE STREQUAL "Release")
+    # Compare logs (NB: this depends on /bin/bash so won't work on non-Unix systems)
```

Comment:
I guess, we check if this is a unix system.
Not sure how to do it.

Reply:
Good point - I've added the check and fixed an issue with the default python version in the update_logfiles script. It should now work